contracts/upgradeable_contracts/RTokenConnector.sol

```diff
@@ -0,0 +1,51 @@
+pragma solidity 0.4.24;
+
+import "openzeppelin-solidity/contracts/AddressUtils.sol";
+import "../interfaces/IRToken.sol";
+import "./Ownable.sol";
+import "./ERC20Bridge.sol";
+
+contract RTokenConnector is Ownable, ERC20Bridge {
+    bytes32 internal constant RTOKEN = 0xdce7cb71f7466970418465f97f1fb7d721c1661bc59b1f0caa7d1f8ce374a47e; // keccak256(abi.encodePacked("rToken"))
+
+    function initializeRToken(
+        address _rToken,
+        address[] _recipients,
+        uint32[] _proportions
+    ) external onlyOwner returns (uint256) {
+        require(AddressUtils.isContract(_rToken));
+        addressStorage[RTOKEN] = _rToken;
+        erc20token().approve(_rToken, uint256(-1));
+        return createAndChangeRTokenHat(_recipients, _proportions);
+    }
+
+    function removeRToken() external onlyOwner {
```

Comment:
should we make sure that the bridge has no balance in rDai before removing it? Does it make sense to redeem all tokens in that case?

Reply:
Good point, I will add it

Reply:
Should we handle the case when something is wrong with `rToken` and `redeemAll` method reverts? Then we won't be able to remove `rToken`

Reply:
should we introduce a flag `force` to handle that case?

Reply:
Good idea, I will add it